<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="welcomeLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>741</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Добро пожаловать!</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="transactionsTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>501</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="newTransaction">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>80</y>
      <width>221</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать транзакцию</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>120</y>
      <width>271</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="amountEdit">
       <property name="placeholderText">
        <string>Сумма</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="descriptionEdit">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Описание</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dateEdit">
       <property name="placeholderText">
        <string>Дата (в формате dd.mm.yyyy)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="labelType">
    <property name="geometry">
     <rect>
      <x>529</x>
      <y>270</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Выберите тип</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>300</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="incomeType">
       <property name="text">
        <string>Доходы</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">typesGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="expensesType">
       <property name="text">
        <string>Расходы</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">typesGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="cancelBtn">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>360</y>
      <width>221</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Отмена</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>460</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="typesGroup"/>
 </buttongroups>
</ui>
